content/md/first-steps/manage-reference-entities.md

```diff
@@ -0,0 +1,233 @@
+To add an attribute, follow these steps:
+1.  Go to `Attributes` tab for a reference entity
+1.  Click on `Add an attribute` button
+1.  Input the label of the attribute in the current catalog locale
+1.  The code is automatically generated using the label but you can update it (this code must be unique and may only contain letters, numbers and underscores)
+1.  Select the attribute type:
+ ![image](../img/ReferenceEntity_AddAttribute_Type.png)
+    - **Text**: for text or text area attributes
+    - **Reference entity single link**: to link a record to only one entity record
+    - **Reference entity multiple links**: to link a record to several entity records
+    - **Single option**: to define a list of options with a single choice
+    - **Multiple options**: to define a list of options with multiple choices
+1.  If the attribute type is a **reference entity single/multiple link**, select the reference entity to link to this attribute in the list
+1.  Tick `Value per locale` if your record could have different values per locale for this attribute
+1.  Tick `Value per channel` if your record could have different values per channel for this attribute
+1.  Click on `Save` button  
+
+The attribute is now created and an edit pop-in opens with the attribute parameters depending on the attribute type.
+
+![image](../img/ReferenceEntity_AddAttribute.png)
+
+:::warning
+These attributes parameters, except the label, can't be edited afterwards.  
+:::
+
+:::info
+To guarantee that the PIM is functional and runs smoothly, we defined a volume limitation of 100 attributes per reference entity for this feature. If you try to create more than 100 attributes, you will have a blocking message.
+:::
+
+## Edit an attribute
+To edit an attribute, follow these steps:
+1.  Go to `Attributes` tab for a reference entity
+1.  Click on `Edit` icon
+1.  An edit pop-in opens with the attribute parameters depending on the attribute type
+1.  You can translate the label of the attribute by switching the current locale in the header
+ ![image](../img/ReferenceEntity_EditAttribute.png)
+1.  Depending on the attribute type, set the specific parameters for the attribute (see below the specific parameters)
+1.  Click on `Save` button
+
+### Edit a text attribute
+ - *Maximum number of characters*: maximum number of characters that an attribute value can have
+ - *Text area*: long text
+
+If `Text area`:
+ - *Rich Text Editor*: activates the WYSIWYG feature to format the text
+
+If not `Text area`:
+ - *Validation rule*: defines how the value should be validated (such as an URL, an email, or using a regular expression)
+
+### Edit an image attribute
+  - *Max file size (MB):* maximum file size allowed in MB
+  - *Allowed extensions*: allowed extensions files (gif, jpeg, jpg, pdf, png...)
+
+### Edit a single/multiple options attribute  
+  - A `Manage options` button to [manage the options of the list](/articles/manage-reference-entities.html#manage-the-options)
```

Comment:
Je dirais "The `Manage options` button enables to manage the options of the list"